webapp/app/forms/steps/steuerlotse_step.py

```diff
@@ -0,0 +1,159 @@
+        return url_for(self.endpoint,
+                       step=step_name,
+                       link_overview=_has_link_overview,
+                       **values)
+
+
+class FormSteuerlotseStep(SteuerlotseStep):
+
+    def __init__(self, title, form, intro=None, endpoint=None, overview_step=None, default_data=None, prev_step=None, next_step=None, header_title=None,
+                 template='basis/form_full_width.html'):
+        super(FormSteuerlotseStep, self).__init__(title, intro, endpoint, overview_step, default_data, prev_step, next_step, header_title)
+        self.form = form
+        self.template = template
+
+    def _pre_handle(self, stored_data):
+        super()._pre_handle(stored_data)
+        form = self.create_form(request, prefilled_data=stored_data)
+        if request.method == 'POST' and form.validate():
+            stored_data.update(form.data)
+        self.render_info.form = form
+        return stored_data
+
+    def _post_handle(self, stored_data):
+        override_session_data(stored_data)
+
+        redirection = self._handle_redirects()
+        if redirection:
+            return redirection
+
+        if request.method == 'POST' and self.render_info.form.validate():
+            return redirect(self.render_info.next_url)
+        return self.render()
+
+    def create_form(self, request, prefilled_data):
+        # If `form_data` is present it will always override `data` during
+        # value binding. For `BooleanFields` an empty/missing value in the `form_data`
+        # will lead to an unchecked box.
+        form_data = request.form
+        if len(form_data) == 0:
+            form_data = None
+
+        form = self.form(form_data, **prefilled_data)
+        return form
+
+    def render(self):
+        """
+        Renders a Form step. Use the render_info to provide all the needed data for rendering.
+        """
+        self.render_info.form.first_field = next(iter(self.render_info.form))
+        return render_template(
+            template_name_or_list=self.template,
+            form=self.render_info.form,
+            render_info=self.render_info,
+            header_title=self.header_title
+        )
+
+    @staticmethod
+    def _delete_dependent_data(data_field_prefixes: list, stored_data: dict):
+        for field in list(stored_data.keys()):
+            if any([field.startswith(data_field_prefix) for data_field_prefix in data_field_prefixes]):
+                stored_data.pop(field)
+        return stored_data
+
+
+class DisplaySteuerlotseStep(SteuerlotseStep):
+
+    def __init__(self, title, intro=None, endpoint=None, overview_step=None, default_data=None, prev_step=None, next_step=None):
+        super(DisplaySteuerlotseStep, self).__init__(title, intro, endpoint, overview_step, default_data, prev_step, next_step)
+
+    def render(self):
+        """
+        Renders a display step. Use the render_info to provide all the needed data for rendering.
+        """
```

Comment:
Because it is not implemented? I think we could still leave it here as a general documentation but reword it a little and talk more about the intention of this function?